### PR TITLE
add cookie-banner

### PR DESCRIPTION
**Pull request by Beck-berry**
Since Matomo is used, we should inform users about tracking, cookie usage. #61 

Banner is added. Link goes to https://www.herrenberg.de/datenschutz .

### CODE REVIEW

File: app/configurations/config.hb.js [hunk 1]
```diff
@@ -349,7 +349,41 @@ export default configMerger(walttiConfig, {
...
+        ],
+        de: [
+          {
+            type: 'text',
+            content:
+              'Diese Seite benutzt Cookies. Wenn Sie die Seite weiterhin benutzten stimmen Sie dem zu. Mehr Informationen: ',
+          },
+          {
+            type: 'a',
+            content: 'Datenshutz',
```
Comment (leonardehrenfried) [inline app/configurations/config.hb.js]: Almost: it's "Datenschutz" (add a c)

**Comment by leonardehrenfried** [comment]
LGTM.

**Comment by hbruch** [comment]
Does Matomo provide a snippet to disable tracking?

**Comment by leonardehrenfried** [comment]
> Does Matomo provide a snippet to disable tracking?

For example by setting a cookie or what do you have in mind?

**Comment by hbruch** [comment]
Yes, this would probably be the implementation.

**Comment by leonardehrenfried** [comment]
It's possible: https://matomo.org/faq/general/faq_157/

But it looks like we need to build our own UI for it.

**Pull request closed without merging**